pkg/validations/common_bucket_validations.go

```diff
@@ -7,8 +7,38 @@ import (
 	"github.com/noobaa/noobaa-operator/v5/pkg/nb"
 	"github.com/noobaa/noobaa-operator/v5/pkg/system"
 	"github.com/noobaa/noobaa-operator/v5/pkg/util"
+	nbv1 "github.com/noobaa/noobaa-operator/v5/pkg/apis/noobaa/v1alpha1"
 )
 
+// ValidateNSFSAccountConfig validates that the provided NSFS config is valid
+func ValidateNSFSAccountConfig(NSFSConfig string) error {
+	log := util.Logger()
+
+	if NSFSConfig == "" {
+		return nil
+	}
+
+	var configObj nbv1.AccountNsfsConfig
```

Comment:
We should update nsfsAccountConfig to comply the newest api - (specifically distinguished_name) - 
https://github.com/noobaa/noobaa-core/blob/master/src/api/common_api.js#L1292

Reply:
and add some more validations for the new addition